vaadin-select-flow-parent/vaadin-select-flow/src/main/java/com/vaadin/flow/component/select/Select.java

```diff
@@ -123,6 +123,10 @@ public Select() {
 
         getElement().setProperty("invalid", false);
         getElement().setProperty("opened", false);
+        // Trigger model-to-presentation conversion in constructor, so that
+        // the client side component has a correct initial value of an empty
+        // string
+        setPresentationValue(null);
```

Comment:
This seems to conflict with the constructor in `GeneratedVaadinSelect` called in line `121`. There you have also a call to setPresentationValue depending on some conditions.
```java
        if ((getElement().getProperty("value") == null
                || !isInitialValueOptional) && initialValue != null) {
            setPresentationValue(initialValue);
        }
```
 Would it be possible to change or remove the conditions in  `GeneratedVaadinSelect` instead of adding this call here?